app/lib/performance_stats.rb

```diff
@@ -0,0 +1,72 @@
+class PerformanceStats
+  def initialize(since, until_days)
```

Comment:
I like keyword arguments as well, but if something is a mandatory argument (and `nil` wouldn't do) I prefer to make it a regular one since then if you try to call the method without it'll break.

Of course, we could set a default to `since` of the last week, and `until_days` is actually redundant as per the TODOs above, since it can be calculated from `since` inside the module.

So yeah, long story short, we'll likely have just one `since:` argument in the next iterations :+1: